tests/helpers/runif.py

```diff
@@ -103,6 +105,7 @@ def __new__(
             tpu: Require that TPU is available.
             ipu: Require that IPU is available.
             hpu: Require that HPU is available.
+            mps: Require that MPS (Apple Sillicon) is available.
```

Comment:
The implementation is different from the others. The way it is right now, mps=False forces the test to only run if mps is absent, if I understand correctly. So the docstring should make that clear

Reply:
yes that’s true and unfortunately necessary. Will adapt the docstrings